node-fs-writefile/cp.js

```diff
@@ -0,0 +1,16 @@
+import { readFile } from 'node:fs/promises';
+import { writeFile } from 'node:fs/promises';
+
+const srcPath= `./${process.argv[2]}`;
+const destPath = `./${process.argv[3]}`;
+
+async function copyFile (src, dest) {
+  try {
+    const srcContent = await readFile(src, 'utf-8');
+    await writeFile(dest, srcContent + '\n');
+  } catch(error) {
+    console.log('failed to copy: ', error.message)
```

Comment:
You usually want to log the entire error, not just the message field. That way you will get a stack trace that will help you figure out where things went wrong.